Model/AccessToken.php

```diff
@@ -40,7 +40,7 @@ class AccessToken
 
     public function __construct(
         string $identifier,
-        DateTimeInterface $expiry,
+        DateTimeImmutable $expiry,
```

Comment:
Why was this changed? It doesn't seem to be related with SF 5.

Reply:
should be reverted.

Reply:
It still hasn't been reverted. Also please don't resolve discussions when you push something.

Reply:
oh ok sorry my bad.

Reply:
should be fixed now :)